### PR TITLE
Additional insights

### PR DESCRIPTION
**Pull request by ps2016-mt**
Added 2 more insights to Julie's to make it 4 insights in total

### CODE REVIEW

File: nba/data_wrangling.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import pandas as pd
+import numpy as np
+from pathlib import Path
+import sqlite3 as sql
+
+# Set the data path (adjust if necessary)
+DATA_PATH = Path.cwd() / "data" / "nba.sqlite"
+
+
+# Data loading function for insights 1 & 2
```
Comment (ty338jy) [inline nba/data_wrangling.py:10]: Hey,
I think load_data, load_data_draft, load_data_team should be the same function, we can just use different table name as parameter?

**Review by ty338jy — APPROVED** [review]
Looks wonderful!

**Review by yuanjingshi — APPROVED** [review]
Good!

File: nba/data_wrangling.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+    SELECT *
+    FROM team_history
+    """
+    df_team = pd.read_sql(query, con)
+    con.close()
+    return df_team
+
+
+# Data wrangling functions
+def drop_empty_str_row(df: pd.DataFrame, col: str) -> pd.DataFrame:
```
Comment (ty338jy) [inline nba/data_wrangling.py:50]: Ideally we would want to keep the docstring, which i think is in the original notebook?

**Pull request merged**